_style/naming-conventions.md

```diff
@@ -32,10 +32,7 @@ This mimics the Java naming convention for classes.
 
 ## Objects
 
-Object names are like class names (upper camel case).
-
-An exception is when mimicking a package or function.
-This isn't common. Example:
+Object should be named in lower camel case :
```

Comment:
I think it's commonplace in the Scala community to always use upper camel case for `object`. I've rarely seen lower camel case for objects, I've only seen it for `package object`s.